docs/src/main/sphinx/admin/properties-write-partitioning.rst

```diff
@@ -0,0 +1,26 @@
+=============================
+Write partitioning properties
+=============================
+
+``use-preferred-write-partitioning``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* **Type:** ``boolean``
+* **Default value:** ``true``
+
+Enable preferred write partitioning. When set to ``true`` and more than the
+minimum number of partitions, set in ``preferred-write-partitioning-min-number-of-partitions``,
+are written, each partition is written by a separate writer. As a result, for some connectors such as the
+Hive connector, only a single new file is written per partition, instead of
+multiple files. Partition writer assignments are distributed across worker
+nodes for parallel processing.
```

Comment:
I would also add
`This can be specified on a per-query basis using the ``use_preferred_write_partitioning`` session property.`

Reply:
These are good suggestions, but I'm simply moving the existing content. Can you open a separate PR with these changes?